### PR TITLE
[ADD] MRP: One-step manufacturing

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR adds a new doc detailing how to manufacture products using one step.

NOTE: There is a ref tag in this doc that does not currently work because the doc it is referencing is not yet live. The doc being referenced should be live by the time this doc is published, at which point, the ref tag will work.

### CODE REVIEW

**Review by Felicious — APPROVED** [review]
Amazing job, @hojo-odoo!

Thank you for addressing my suggestions, the doc looks flawless to me :smile:

Warm regards,
Felicia

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+======================
+
+.. _manufacturing/management/one_step_manufacturing:
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+Odoo *Manufacturing* allows users to manufacture products using one, two, or three steps. When using
+one-step manufacturing, Odoo creates a manufacturing order, but does not generate transfers for the
+movement of components out of inventory or finished products into stock. Inventory counts still
+update based on the number of components used and products manufactured, but the act of transferring
+them to and from inventory is not tracked.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:12]:
```suggestion
Odoo *Manufacturing* allows users to manufacture products using one, two, or three steps.

When using one-step manufacturing, Odoo creates a manufacturing order, but does **not** generate
transfers for the movement of components out of inventory, or finished products into stock.

Inventory counts still update based on the number of components used and products manufactured, but
the act of transferring them to and from inventory is **not** tracked.
```

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+.. tip::
+   The number of steps used in manufacturing is set at the warehouse level, allowing for each
+   warehouse to use a different number of steps. To change the number of steps used for a specific
+   warehouse, begin by navigating to :menuselection:`Inventory --> Configuration --> Warehouses`,
+   and then select a warehouse from the :guilabel:`Warehouses` screen.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:18]:
```suggestion
   then select a warehouse from the :guilabel:`Warehouses` screen.
```

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+   On the :guilabel:`Warehouse Configuration` tab, find the :guilabel:`Manufacture` radio input
+   field and select one of the three options: :guilabel:`Manufacture (1 step)`, :guilabel:`Pick
+   components and then manufacture (2 steps)`, or :guilabel:`Pick components, manufacture and then
+   store products (3 steps)`.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:23]:
```suggestion
   field, and select one of the three options: :guilabel:`Manufacture (1 step)`, :guilabel:`Pick
   components and then manufacture (2 steps)`, or :guilabel:`Pick components, manufacture and then
   store products (3 steps)`.
```

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+--------------
+
+To complete work orders from the manufacturing order itself, begin by navigating to
+:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, then select a manufacturing
+order.
+
+On the manufacturing order page, select the :guilabel:`Work Orders` tab. Once work begins on the
+first work order that needs to be completed, click the :guilabel:`Start` button for that work order.
+Odoo *Manufacturing* then starts a timer that keeps track of how long the work order takes to
+complete.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:71]:
```suggestion
Odoo *Manufacturing* then starts a timer to keep track of how long the work order takes to complete.
```

**Review by ksc-odoo — APPROVED** [review]
@hojo-odoo Great work on this doc. Only had a handful of minor adjustments that need your attention -- but once those are addressed, I think this will be ready for the next stage. Nicely done! 👍

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+Clicking :guilabel:`Mark as Done` while there is at least one more work order left to complete opens
+a page that lists the next work order. Click on that work order to open it in the tablet view.
+
+Once the final work order for the |MO| has been reached, a :guilabel:`Mark as Done and Close MO`
+button appears on the tablet view in addition to the :guilabel:`Mark as Done` button. Click
+:guilabel:`Mark as Done and Close MO` to mark the |MO| as :guilabel:`Done` and register the
+manufactured product(s) into inventory.
+
+.. image:: one_step_manufacturing/done-and-close.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:118]: this isn't very useful without the surrounding UI as navigational context

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+.. image:: one_step_manufacturing/tablet-view-button.png
+   :align: center
+   :alt: The tablet view button for a work order on a manufacturing order.
+
+After opening the tablet view, Odoo *Manufacturing* automatically starts a timer that keeps track of
+how long the work order takes to complete. After completing the work order, click the
+:guilabel:`Mark as Done` button in the top-right corner of the tablet view.
+
+.. image:: one_step_manufacturing/mark-as-done.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:106]: same here
would consider removing

**Review by StraubCreative — APPROVED** [review]

**Comment by StraubCreative** [comment]
@robodoo delegate=hojo-odoo

**Comment by hojo-odoo** [comment]
r+

**Comment by hojo-odoo** [comment]
@robodoo r+

**Comment by hojo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**